addons/hr_homeworking_calendar/wizard/homework_location_wizard.xml

```diff
@@ -10,8 +10,9 @@
                     <div class="o_row w-100">
                         <field name="date"/>
                     </div>
-                    <label for="weekly"/>
-                    <div class="o_row w-100">
+                    <field name="user_can_edit" invisible="1"/>
```

Comment:
seems not usefull given the last changes of the framework
https://github.com/odoo/odoo/pull/137031